processor/processor.go

```diff
@@ -2160,23 +2190,64 @@ func (proc *HandleT) pipelineWithPause(ctx context.Context, fn func(ctx context.
 	}
 }
 
+//`jobSplitter` func Splits the read Jobs into sub-batches after reading from DB to process.
+//`subJobMerger` func merges the split jobs into a single batch before writing to DB.
+//So, to keep track of sub-batch we have `hasMore` variable.
+//each sub-batch has `hasMore`. If, a sub-batch is the last one from the batch it's marked as `false`, else `true`.
+type subJobT struct {
```

Comment:
lets avoid the `T` suffix

```suggestion
type subJob struct {
```